File: client/src/app/_models/userParams.ts
import { PaginationParams } from "./paginationParams";
import { User } from "./user";

export class UserParams extends PaginationParams {
  gender = "female";
  minAge = 18;
  maxAge = 150;
  orderBy = "lastActive";
  constructor(user: User) {
    super()
    // futur filter
  }
}

File: API/Helpers/UserParams.cs
namespace API.Helpers;

public class UserParams : Pagination
{
	public string CurrentUsername { get; set; }
	public int MinAge { get; set; } = 18;
	public int MaxAge { get; set; } = 150;
	public string gender { get; set;}
	public string OrderBy { get; set; } = "lastActive";
}
